internals/reminders_test.py

```diff
@@ -37,18 +37,39 @@ def __init__(self, status_code=200, text='{}'):
 
 
 def make_test_features():
-  feature_1 = core_models.Feature(
-      name='feature one', summary='sum', owner=['feature_owner@example.com'],
-      category=1, ot_milestone_desktop_start=100)
+  feature_1 = FeatureEntry(
+      name='feature one', summary='sum',
+      owner_emails=['feature_owner@example.com'],
+      category=1, feature_type=0)
   feature_1.put()
-  feature_2 = core_models.Feature(
+  stages = [110, 120, 130, 140, 150, 151, 160]
```

Comment:
Should we use the core_enum variables here instead of `[110, 120, 130, 140, 150, 151, 160]`?

Also is there a reason to use the incubating stages here vs feature_entry 2 uses the existing id stages?

Optional suggestion:
- `[incubatingStage[0] for incubatingStage in STAGES_AND_GATES_BY_FEATURE_TYPE[FEATURE_TYPE_INCUBATE_ID]]` instead of `[110, 120, 130, 140, 150, 151, 160]`
- Why: I had to look to do a global lookup of 110 to find it comes from core_enums. It'll make it easier to track down valid values of this.